### DISCUSSION TITLE
Version with all addons?

### DISCUSSION THREAD

**Discussion by Explosion-Scratch** [ScratchAddons/ScratchAddons#555] [Ideas] [answered]
**Is your feature request related to a problem? Please describe.**

I like some of the addons that have been removed. 

**Describe the solution you'd like**

A branch that addons are never removed from. This would be really cool.

**Describe alternatives you've considered**

No.
**Additional context**

I really liked the data category tweaks and I don't really care about the bugs, I'll use it anyways.

**Comment by GrahamSH-LLK** [comment]
Maybe that addon could be a hidden "Easter Egg"? In the future, that addon could be re-added if it worked.

**Comment by WorldLanguages** [comment, marked as answer]
For now, I suggest you add it manually locally. Removed addons will be forever stored in git history.

**Comment by Explosion-Scratch** [comment]
Okay!

**Comment by cobaltt7** [comment]
Scratch Addons Nightly now exists (@lisa-wolfgang)

**Reply by Dhuls-Scratch** [reply]
Where can I find it?

**Reply by Norbiros** [reply]
You can go on our discord, or here https://github.com/lisa-wolfgang/ScratchAddons/tree/sa-nightly

**Reply by cobaltt7** [reply]
The Discord just links the fork?
